backend/src/main/java/com/roc/app/match/dto/UserMatchDto.java

```diff
@@ -0,0 +1,31 @@
+package com.roc.app.match.dto;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+
+import java.time.LocalDateTime;
+
+public record UserMatchDto(
```

Comment:
```suggestion
public record ParticipantMatchResponseDto(
```
Verify with the database. Some field may be null, as we'll be showing the participant matches, that didn't occur yet. Then, we won't know the score or winner. In the beginning, we won't even know the opponent. Please adjust accordingly.